tests/components/fritzbox/test_light.py

```diff
@@ -56,9 +59,11 @@ async def test_setup(hass: HomeAssistant, fritz: Mock) -> None:
     assert state
     assert state.state == STATE_ON
     assert state.attributes[ATTR_FRIENDLY_NAME] == "fake_name"
+    assert state.attributes[ATTR_COLOR_MODE] == ColorMode.COLOR_TEMP
     assert state.attributes[ATTR_COLOR_TEMP_KELVIN] == 2700
     assert state.attributes[ATTR_MIN_COLOR_TEMP_KELVIN] == 2700
     assert state.attributes[ATTR_MAX_COLOR_TEMP_KELVIN] == 6500
+    assert state.attributes[ATTR_HS_COLOR] == (28.395, 65.723)
```

Comment:
Where do these hs values come from?

Reply:
they are automatically calculated from the color temp by the `LightEntity` class in case `hs_color` property returns `None`

Reply:
Ah, this is calculated from the color temperature. Makes sense, I did not know that.